CorsixTH/Lua/entities/patient.lua

```diff
@@ -23,6 +23,7 @@ class "Patient" (Humanoid)
 
 ---@type Patient
 local Patient = _G["Patient"]
+local go_home_reasons = {CURED = "cured", KICKED = "kicked", OVER_PRICED = "overpriced"}
```

Comment:
I guess this makes sense, the "enum" only serves to self-document the reasons for going home, annotating the method would be fine then.